### PR TITLE
Switch to Esp32

### LINKED ISSUE #21: Over the air update
The user should be able to update the firmware without having to connect the frame to a computer

### PR DESCRIPTION
**Pull request by MoJo2600**
Switching to ESP32 as we had to many issues with the ESP8266

Working:
* OTA
* Website and basic features

Not working:
* Configuration Endpoint

fixes #21

### CODE REVIEW

**Review by MoJo2600 — COMMENTED** [review]
Lets Go To The Moon!

**Review by christofwe — APPROVED** [review]
love it

File: config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (KriegerTobias) [inline config.json]: this file should probably not be in here, at least not with this content

**Review by KriegerTobias — APPROVED** [review]
looking good 🎉 
for the one file you could have a look before merging

File: config.json (hunk 1 above)
Comment (MoJo2600) [reply config.json]: Aaaaand... it's gone

**Pull request merged**